doc/nrf/device_guides/working_with_nrf/nrf70/developing/stack_combo.csv

```diff
@@ -0,0 +1,9 @@
+Stack components,Wi-Fi SSID scanning with cellular backhaul,Wi-Fi application for end-device-to-cloud products,Wi-Fi + BLE application for end-device-to-cloud products,Matter over Wi-Fi products,Matter over Thread products,Dual-stack capable Matter devices (not gateways)
+Wi-Fi scan only driver,X,,,,,
+Wi-Fi full stack ,,X,X,X,,X
```

Comment:
instead of full stack (which is a rather ambiguous) should we have "Wi-Fi client stack"?